### PR TITLE
Use transaction-generator-lib in executor benchmark

### PR DESCRIPTION
**Pull request by igor-aptos**
enabling to test on same workloads as Forge

### CODE REVIEW

File: execution/executor-benchmark/src/gen_executor.rs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{time::Duration, sync::atomic::AtomicUsize};
+use aptos_types::transaction::{SignedTransaction, Transaction};
+use aptos_types::account_address::AccountAddress;
+use aptos_transaction_generator_lib::TransactionExecutor as GenInitTransactionExecutor;
```
Comment (msmouse) [inline execution/executor-benchmark/src/gen_executor.rs:8]: What does  "gen init" mean in this context? 😂

**Review by msmouse — APPROVED** [review]
(seems legit)

File: execution/executor-benchmark/src/db_access.rs [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    language_storage::{StructTag, TypeTag},
+};
+use once_cell::sync::Lazy;
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
+use std::str::FromStr;
+
+// Note: in case this changes in the future, it doesn't have to be a constant, and can be read from
+// genesis directly if necessary.
+pub static TOTAL_SUPPLY_STATE_KEY: Lazy<StateKey> = Lazy::new(|| {
+    StateKey::table_item(
```
Comment (JoshLind) [inline execution/executor-benchmark/src/db_access.rs:21]: Scary 😨

**Review by JoshLind — APPROVED** [review]
Unblocking

File: execution/executor-benchmark/src/db_access.rs (hunk 2 above)
Comment (igor-aptos) [reply execution/executor-benchmark/src/db_access.rs:21]: @grao1991's code, just extracted to a separate file here :)

File: execution/executor-benchmark/src/gen_executor.rs (hunk 1 above)
Comment (igor-aptos) [reply execution/executor-benchmark/src/gen_executor.rs]: TransactionExecutor for transaction generator initialization. I didn't think of naming conflict when I was naming it that, maybe  aptos_transaction_generator_lib::TransactionExecutor should be renamed to something like ClientTransactionExecutor or ReliableTransactionSubmitter

**Pull request merged**